src/components/Form/СreateUserAccountForm/CreateUserAccountForm.tsx

```diff
@@ -0,0 +1,108 @@
+import React, { useMemo, useState } from 'react';
+import { Box, TextField, Typography } from '@material-ui/core';
+import i18n from 'i18next';
+import { langTokens } from '../../../locales/localizationInit';
+import { UserAccountButton } from '../Buttons';
+import { useStyles } from './styles/CreateUserAccountForm.style';
+import { regex } from '../../../views/Profile/PersonalInfo/constants/regex';
+import { UserEmailType } from '../../../views/Profile/PersonalInfo/types';
+
+type CreateUserAccountFormPropsType = {
+  usersEmails: UserEmailType;
+  onClick: (
+    e: React.MouseEvent<HTMLButtonElement, MouseEvent>,
+    email: string,
+  ) => void;
+  currentUserEmail?: string;
+  isLoading?: boolean;
+};
+type ErrorFieldsType = {
+  email: string;
+};
+type VisitedFieldsType = {
+  email: boolean;
+};
+export const CreateUserAccountForm: React.FC<CreateUserAccountFormPropsType> = ({
+  onClick,
+  usersEmails,
+  currentUserEmail,
+  isLoading,
+}) => {
+  const [userEmail, setUserEmail] = useState('');
+  const [visitFields, setVisitFields] = useState<VisitedFieldsType>({
+    email: false,
+  });
+
+  const classes = useStyles();
+  const emailChangeHandler = (
+    e: React.ChangeEvent<HTMLInputElement | HTMLTextAreaElement>,
+  ) => {
+    setUserEmail(e.target.value);
+  };
+
+  const errorMessage: ErrorFieldsType = useMemo(() => {
+    const errors: ErrorFieldsType = {
+      email: '',
+    };
+
+    if (currentUserEmail) {
```

Comment:
could you please clarify this particular condition? so if prop `currentUserEmail` is passed from the outside, we immediately show the error message?

Reply:
If user has currentEmail - it means that he had already sent his email to activate his acc. So in meeting, we discussed that should be displayed error in this case just to not allow user to send new email, because he need to confirm old